### PR TITLE
14405 Prep for PDF file input

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#14405

*Description of changes:*
- now save user KC GUID to store
- added PageSizes enum
- renamed ApiDocumentsIF -> FetchDocumentsIF
- added Pdf Info interface
- added Presigned Url interface
- added isTypeCourtOrder to Enum Mixin
- added some typings to services
- small cleanup/comments
- updated unit tests
- added PDF JS package
- added Volar-Vetur package + config
- app version = 5.9.0

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "5.8.6",
+  "version": "5.9.0",
```
Comment (severinbeauvais) [inline package.json:3]: This is the start of a new feature so it makes sense to update the minor release number.

File: package.json [hunk 2]
```diff
@@ -33,6 +33,7 @@
     "http-status-codes": "^2.2.0",
     "launchdarkly-js-client-sdk": "^2.22.1",
     "lodash": "^4.17.21",
+    "pdfjs-dist": "2.0.943",
```
Comment (severinbeauvais) [inline package.json:36]: Will be needed soon for PDF validation.

File: package.json [hunk 3]
```diff
@@ -54,6 +55,7 @@
     "@types/vuelidate": "^0.7.15",
     "@typescript-eslint/eslint-plugin": "^5.38.0",
     "@typescript-eslint/parser": "^5.38.0",
+    "@volar-plugins/vetur": "latest",
```
Comment (severinbeauvais) [inline package.json:58]: Should help devs using VS Code Vue Language Features (Volar) extension.

File: src/interfaces/form-interface.ts [hunk 4]
```diff
@@ -1,5 +1,14 @@
...
 export interface FormIF extends HTMLFormElement {
+  /** Will clear all inputs and reset their validation errors */
   reset(): void
+
+  /** Will only reset input validation and not alter their state. */
   resetValidation(): void
+
+  /** Will validate all inputs and return if they are all valid or not. */
   validate(): boolean
 }
```
Comment (severinbeauvais) [inline src/interfaces/form-interface.ts:14]: I made these updates because I'm leveraging this interface for the new PDF file input component (my next PR).

**Review by vysakh-menon-aot — APPROVED** [review]

**Pull request merged**